SPECS/nvme-cli/nvme-cli.spec

```diff
@@ -22,14 +26,26 @@ make CFLAGS="%{build_cflags} -std=gnu99 -I."
 %install
 make install PREFIX=%{_prefix} DESTDIR=%{buildroot}
 
+%check
+pip3 install nose nose2 pep8 flake8 pylint epydoc
```

Comment:
Is it okay to install non-mariner packages for check purposes?

Reply:
Yes, we use pip to install test dependencies that are not available in mariner. There are quite a few specs doing the same. None of the external packages are shipped; just using for the check tests.